### ISSUE TITLE
Chartist error when given an empty dataset

### ISSUE THREAD

**Issue by ilanbiala** [chartist-js/chartist#175]
When given a dataset with 0's, I get an error:

```
Uncaught TypeError: Cannot read property 'units' of undefined chartist.min.js:7
```

With this code:

```
var data = {
  series: ['Test point', 'Another test point']
  labels: [[0, 0]]
};
```

**Comment by gionkunz** [comment]
Your data object is wrong. Series should contain arrays of arrays.

``` javascript
var data = {
  series: [[0, 0]]
  labels: ['Test point', 'Another test point']
};
```

Please check the examples in the API documentation on how to fill the data object.

**Comment by ilanbiala** [comment]
@gionkunz, sorry about that. I have a lot more data and I was just keeping it brief and I missed the wrapping array. I didn't get this error with 0.6.1, so it's something new in 0.7.0.

**Comment by gionkunz** [comment]
Can you post a full example that reproduces the issue?

**Comment by gionkunz** [comment]
This works for me: 
http://jsbin.com/xikuta/1/edit?js,console,output

Please post the full code that causes this issue.

Cheers

**Comment by gionkunz** [comment]
Hi this was infact a bug in 0.7.0 which makes the release useless :-/ Ive published 0.7.1. Cheers

**Comment by ilanbiala** [comment]
Thanks!
